elliptic-curve/Cargo.toml

```
@@ -49,7 +49,7 @@ alloc = ["der/alloc", "sec1/alloc", "zeroize/alloc"] # todo: use weak activation
 arithmetic = ["ff", "group"]
 bits = ["arithmetic", "ff/bits"]
 dev = ["arithmetic", "hex-literal", "pem", "pkcs8"]
-hash2curve = ["digest", "ff", "group"]
+hash2curve = ["arithmetic", "digest", "ff", "group"]
```

```suggestion
hash2curve = ["arithmetic", "digest"]
```

The `arithmetic` feature already pulls in `ff` and `group`

---

Done.